util/dvsim/SimCfg.py

```diff
@@ -392,8 +392,11 @@ def prune_items(items, marked_items):
             if self.reseed_ovrd is not None:
                 test.reseed = self.reseed_ovrd
 
-            # Apply reseed multiplier if set on the command line.
-            test.reseed *= self.reseed_multiplier
+            # Apply reseed multiplier if set on the command line. This is
+            # always positive but might not be an integer. Round to nearest,
+            # but pick at least one.
+            scaled = int(test.reseed * self.reseed_multiplier + 0.5)
```

Comment:
nit: I'd go for `math.ceil()`.

Reply:
Good point, although this is supposed to be `round()`: a strong argument for using a named function!